packages/apollo-server/tsconfig.json

```diff
@@ -0,0 +1,15 @@
+{
+  "extends": "../../tsconfig",
+  "compilerOptions": {
+    "rootDir": "./src",
+    "outDir": "./dist",
+    "target": "es6",
```

Comment:
Top level has target es5. Why is this package special? Do we intend to require Node 8?

Reply:
Node.js 6.x >= 6.5 is [99% ES6 complete](https://node.green/) without `--harmony`.

Given that, if our Node.js 6 tests pass, I think we can compile to `es6` (ES2015) and be comfortable declaring support for Node 6.  Earlier Node.js ES6 was incomplete or flagged behind `--harmony`, but I'd expect Node.js 6 users have mostly updated to more recent versions by now since 6.14.1 is the most recent version.

Node 4 will be toast on Monday. 🍞🙇🐟

Reply:
That said, if you have concerns @glasser we can certainly stay at `es5`.  This is server-land, after all: _Where the ECMAScript support is (mostly) made up and the bytes don't matter.™_